src/components/ResourceList/components/FilterControl/components/DateSelector/tests/DateSelector.test.tsx

```diff
@@ -161,6 +161,32 @@ describe('<DateSelector />', () => {
     });
   });
 
+  describe('timezones adjustments', () => {
+    const timeZoneOffsets = [720, 540, 60, 0, -60, -540, -720];
+    const date = '2019-01-01';
+    const getTimezoneOffset = Date.prototype.getTimezoneOffset;
+
+    timeZoneOffsets.forEach((zone) => {
```

Comment:
I would also 🔥 this `forEach` here, if it works for one offset it should work for all offsets no?
Assuming your mock is properly called